ecf/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.ecf

```diff
@@ -34,7 +34,7 @@ module load prod_envir/${prodenvir_ver}
 
 module load geos/${geos_ver}
 module load proj/${proj_ver}
-module load libfabric/${libfabric}
+module load libfabric/${libfabric_ver}
```

Comment:
Please test the following recommendation:
1- replace `module purge` with `module reset`
2- remove line 37
By using module reset instead of module purge you will no longer need to load the libfabric module.